### PR TITLE
Fix broken icon alternative solution

### PR DESCRIPTION
**Pull request by wangqiang1208**
@jsyeo This is an alternative solution, I uploaded the picture to s3. 

This won't break @jiunwei-moe 's test cases but need to run the rake script.

### CODE REVIEW

**Comment by jiunwei-moe** [comment]
@wangqiang1208 Thanks for working around the test case. The test case was included as the program used to generate PDFs doesn't work with path-relative URLs. If it's not important for images to show up correctly in PDFs, either this or #364 may be accepted.

For the general case, I'll work on a patch to use `wicked_pdf_image_tag` where appropriate so the correct image URL will be used if a PDF is being generated.

**Comment by wangqiang1208** [comment]
@jiunwei-moe Thanks!  I think we should not break the images in pdf. 

Yes, for general case you should do that, as far as I know, our course logo uses relative path.

**Comment by wangqiang1208** [comment]
@jsyeo  I think we'd better use this approach ?

File: app/models/user.rb [hunk 1]
```diff
@@ -116,7 +116,7 @@ def self.new_with_session(params, session)
 
   def use_default_photo_pic?
     self.profile_photo_url ==
-        'https://fbcdn-profile-a.akamaihd.net/hprofile-ak-ash4/c178.0.604.604/s160x160/252231_1002029915278_1941483569_n.jpg'
+        'http://coursemology.s3.amazonaws.com/public/default_profile_pic.png'
```
Comment (jsyeo) [inline app/models/user.rb]: hey @wangqiang1208, this isn't very DRY. It's repeated twice. Do you think you can extract this somewhere?

File: app/models/user.rb [hunk 2]
```diff
@@ -34,6 +34,8 @@ class User < ActiveRecord::Base
 
   belongs_to :system_role, class_name: "Role"
 
+  DEFAULT_PROFILE_PIC_URL = 'http://coursemology.s3.amazonaws.com/public/default_profile_pic.png'
```
Comment (wangqiang1208) [inline app/models/user.rb]: @jsyeo I changed to use constants leh.

Comment (jsyeo) [inline app/models/user.rb]: :+1:

> @jsyeo I changed to use constants leh.

By the way, this is not the right place to use *leh*. :stuck_out_tongue: Just saying. heh.

lol

Comment (wangqiang1208) [inline app/models/user.rb]: Hmm..actually I searched for a while and didn't find the answer, so normally where is the correct place ?

Comment (jsyeo) [inline app/models/user.rb]: I wasn't referring to that. Uh.. nevermind.

Comment (wangqiang1208) [inline app/models/user.rb]: Ah, haha, I understand 'leh' !

**Pull request merged**